app.rb

```diff
@@ -10,9 +10,12 @@ class App < Sinatra::Base
   register Sinatra::SensibleLogging
 
   sensible_logging(
-    logger: Logger.new(STDOUT)
+    logger: Logger.new(STDOUT),
+    log_tags: [->(req) { req.body.read }]
   )
 
+  before { request.body.rewind }
```

Comment:
in terms of ordering, does this block execute before or after the middleware is called?

Reply:
This executes after the middleware, before the request hits a route

Reply:
@cybojenix I've gone for the solution we initially discussed.
Your question made me realise that depending on implicit ordering is actually pretty awful.